setup.py

```diff
@@ -10,7 +10,7 @@
     author="Department for Levelling Up, Housing and Communities",
     description="Provides access to functionality common to creating a data dashboard.",
     name="gov_uk_dashboards",
-    version="7.2.1",
+    version="7.3.1",
```

Comment:
Being a pedant here - it's not really a problem, but this should have been 7.3.0